wbooks/src/app/components/BookList/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FlatList, ListRenderItem } from 'react-native';
+import { IBook } from '@interfaces/BookInfo';
+import BookCard from '@components/BookCard';
+
+import styles from './styles';
+
+interface IProps {
+  bookArray: IBook[];
+}
+
+function BookList(props: IProps) {
+  const renderItem: ListRenderItem<IBook> = ({ item }: { item: IBook }) => <BookCard book={item} />;
```

Comment:
Try this: `const renderItem: ListRenderItem<IBook> = ({ item }) => <BookCard book={item} />;`

Reply:
Yes, it becomes redundant to point at `IBook` interface in the `renderItem` function and the argument type `Item`. 
Updated.